### DISCUSSION TITLE
What's different between `Observale.ToBinding()` and `new Binding()`

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#13501] [Q&A] [answered]
I saw the docs
https://docs.avaloniaui.net/zh-Hans/docs/next/guides/data-binding/binding-from-code#%E5%9C%A8%E5%AF%B9%E8%B1%A1%E5%88%9D%E5%A7%8B%E5%8C%96%E5%99%A8%E4%B8%AD%E8%AE%BE%E7%BD%AE%E7%BB%91%E5%AE%9A
and
https://docs.avaloniaui.net/zh-Hans/docs/next/guides/data-binding/binding-from-code#%E4%BB%8E%E4%BB%A3%E7%A0%81%E4%B8%AD%E4%BD%BF%E7%94%A8-xaml-%E7%BB%91%E5%AE%9A
What it's better to use that for `Binding`?
Or both have the same effect?

If use `AOT`, what is the best choise?

**Comment by maxkatz6** [comment]
`Observale.ToBinding()` just wraps existing Observable into the Binding. There is no AOT complications here, unless your observable itself is produced from reflectiom based source.

`new Binding()` is reflection-based binding, as per usual.

**Reply by Coloryr** [reply]
Is mean if `AOT`, use `new Binding()` is work, but use `Observale.ToBinding()` is better?

**Reply by maxkatz6** [reply, marked as answer]
`new Binding(string)` accesses members via reflection, and it's not type-checked. Observable is generally would be better for both AOT and type-safety, yes.

**Reply by Coloryr** [reply]
Thanks
